test/unit/test_threaded_downloader.py

```diff
@@ -164,7 +164,7 @@ def test_download_unhandled_exception(self):
 
             self.assertTrue(downloader.is_canceled)
 
-            expected_log_message = 'Unhandled Exception in Worker Thread'
+            expected_log_message = 'Attempting to connect to'
```

Comment:
I'm not really sure if I done the right thing here? The old code didn't appear to do anything?